packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -84,6 +88,22 @@ export default class {
     return handler.test()
   }
 
+  async getAllRemotesInfo() {
+    const remotes = await this._remotes.get()
+    return asyncMap(remotes, async remote => {
+      try {
+        const info = await timeout.call(
+          this.getRemoteInfo(remote.id),
+          DEFAULT_TIMEOUT
+        )
+        this._remotesInfo = { ...obfuscateRemote(remote), info }
+        return this._remotesInfo
+      } catch (error) {
+        return this._remotesInfo
+      }
+    })
+  }
```

Comment:
This implementation looks really weird, have you tested it with `xo-cli`?

If you did and you think it's working fine, please paste me the result here.

Reply:
```sh
$ xo-cli remote.getAllInfo
[
  {
    "name": "test",
    "enabled": true,
    "error": "",
    "id": "29e8ec99-9e7c-472e-9f39-83dc5e30d0d6",
    "url": "file:///tmp/bkp",
    "info": {
      "filesystem": "tmpfs",
      "size": 8378896384,
      "used": 13443072,
      "available": 8365453312,
      "capacity": 0.01,
      "mountpoint": "/tmp"
    }
  },
  {
    "name": "testSMB",
    "enabled": true,
    "error": "",
    "id": "ee914cb3-8d4a-4832-8144-8a4708596277",
    "url": "smb://login:pass@WORKGROUP\\\\ip\\smb\u0000",
    "info": {}
  }
]
```

Reply:
It appears correct (except I only want the info) but your cache does not work.